### PR TITLE
Modifications to the message popup

### PR DESCRIPTION
**Pull request by apenalver2**
- Include a customizable icon with icon class and icon properties class
- Customize the class of the buttons

ISSUE #192

### CODE REVIEW

File: src/app/showcase/components/message-popup/showcase-message-popup.component.ts [hunk 1]
```diff
@@ -34,4 +35,15 @@ export class ShowcaseMessagePopupComponent {
 				console.log('Observable returned to showcase', v);
 			});
 	}
+
+	public showCustomized() {
```
Comment (dskuratovich) [inline src/app/showcase/components/message-popup/showcase-message-popup.component.ts]: There's no need to mark this as public, it's set by default

File: src/app/showcase/components/message-popup/showcase-message-popup.component.ts [hunk 2]
```diff
@@ -34,4 +35,15 @@ export class ShowcaseMessagePopupComponent {
...
+		const buttons: MessagePopupButton[] = [];
```
Comment (dskuratovich) [inline src/app/showcase/components/message-popup/showcase-message-popup.component.ts]: You can easily define all the items of the array on initializing stage

File: src/app/showcase/components/message-popup/showcase-message-popup.component.ts (hunk 1 above)
Comment (apenalver2) [reply src/app/showcase/components/message-popup/showcase-message-popup.component.ts]: Ok, thanks for the advice

File: src/app/showcase/components/message-popup/showcase-message-popup.component.ts (hunk 2 above)
Comment (apenalver2) [reply src/app/showcase/components/message-popup/showcase-message-popup.component.ts]: thanks, done

File: src/app/showcase/components/message-popup/showcase-message-popup.component.html [hunk 3]
```diff
@@ -11,6 +11,8 @@
     <button type="button" class="btn mt-2 mr-1" (click)="showQuestionYN();"><i
             class="icon-question-circle mr-1"></i>Question YN
     </button>
+    <button type="button" class="btn mt-2 mr-1" (click)="showCustomized();">Customized
```
Comment (alfonsserra) [inline src/app/showcase/components/message-popup/showcase-message-popup.component.html]: Non necessary ;

Comment (apenalver2) [reply src/app/showcase/components/message-popup/showcase-message-popup.component.html]: true, I will fix also the other buttons click functions

**Pull request merged**